crates/circuit/src/duration.rs

```diff
@@ -70,14 +72,17 @@ impl Duration {
     fn py_value(&self, py: Python) -> PyResult<PyObject> {
         match self {
             Duration::dt(v) => v.into_py_any(py),
-            Duration::us(v) | Duration::ns(v) | Duration::ms(v) | Duration::s(v) => {
-                v.into_py_any(py)
-            }
+            Duration::ps(v)
+            | Duration::us(v)
+            | Duration::ns(v)
+            | Duration::ms(v)
+            | Duration::s(v) => v.into_py_any(py),
         }
     }
 
     fn __repr__(&self) -> String {
         match self {
+            Duration::ps(t) => format!("Duration::ps({t})"),
```

Comment:
Should this be?
```suggestion
            Duration::ps(t) => format!("Duration.ps({t})"),
```
Since this is the Python repr, also all the others use `.`.

Reply:
Fixed in 914fb73a76079d7efe5722516687b956d5381fbd